### ISSUE TITLE
submodules appear to break my fame

### ISSUE THREAD

**Issue by schultz9999** [oleander/git-fame-rb#59]
I have a submodule and getting an error for its directory. Any hints? Thanks!
```
▶ git fame

Error: Could not run 'git --git-dir='/x/y/z/.git' --work-tree='/x/y/z' blame --encoding=UTF-8 -p -M --date=local master  -- 'my-submodule'' => fatal: no such path my-submodule in master.
Try --help for help.
```

**Comment by psalaberria002** [comment]
You can use --exclude /path/to/submodule for excluding the submodules.

**Comment by oleander** [comment]
Thanks for your bug report. This has been fixed in `2.4.0` and can be installed with `gem install git_fame --version 2.4.0`.
